### ISSUE TITLE
DATA_COLLECTION_PERMISSIONS_PROP_RESERVED

### ISSUE THREAD

**Issue by regseb** [mozilla/addons-linter#5845]
When you add the `data_collection_permissions` property in `manifest.json`, addons-linter reports the error: _The "data_collection_permissions" property is reserved._ (`DATA_COLLECTION_PERMISSIONS_PROP_RESERVED`)

But when you publish an extension without the `data_collection_permissions` property, you get the warning:

> **The "data_collection_permissions" property is missing.**
>
> **Avertissement:** The "/browser_specific_settings/gecko/data_collection_permissions" property is required for all new Firefox extensions, and will be required for new versions of existing extensions in the future. Please add this key to the manifest. More information at: <https://mzl.la/firefox-builtin-data-consent>

**Comment by willdurand** [comment]
I am guessing that's because you're passing your own `config`, which overrides the entire `config` object in the linter. In most cases, this is fine, except for features that are gated by a config flag. You could add `enableDataCollectionPermissions: true` as follows:

```js
import linter from "addons-linter";

await linter.createInstance({
    config: {
        _: [process.argv[2]],
        logLevel: "debug",
        enableDataCollectionPermissions: true,
    },
}).run();
```
